libs/community/langchain_community/vectorstores/bageldb.py

```diff
@@ -109,6 +109,12 @@ def __query_cluster(
             import bagel  # noqa: F401
         except ImportError:
             raise ImportError("Please install bagel `pip install betabageldb`.")
+
+        texts = list(query_texts)
```

Comment:
will raise errors if  query_texts is None

Reply:
Thank you for the review. You are correct. I have addressed this issue in a subsequent commit. Would you prefer to have these two commits squashed into one, or is it fine to keep them as two separate commits?